contracts/lbp/LBPManager.sol

```diff
@@ -302,6 +349,7 @@ contract LBPManager {
      * @param _swapEnabled              Enables/disables swapping.
      */
     function setSwapEnabled(bool _swapEnabled) external onlyAdmin {
+        require(TaskState.Ended != state, "LBPManager: LBP ended");
```

Comment:
Why?  you can only call this function if the LBP is ended?  For what purpose?

Reply:
You can call this method only if the LBP is not ended. So that nobody can restart swapping after end time.

Reply:
Improved the condition, and is now not dependent on state.